src/lib.rs

```diff
@@ -59,9 +59,33 @@ struct FlatComrakOptions {
   extension_tagfilter: bool,
   extension_tasklist: bool,
   #[serde(default)]
+  extension_multiline_block_quotes: bool,
+  #[serde(default)]
+  extension_alerts: bool,
+  #[serde(default)]
+  extension_math_dollars: bool,
+  #[serde(default)]
+  extension_math_code: bool,
+  #[serde(default)]
+  extension_wikilinks_title_after_pipe: bool,
```

Comment:
I don't particularly like how comrak exposes this config option... it seems silly to me to have two options (before_pipe, after_pipe) for something that can be accomplished with one. If we followed the same pattern used on this option, then (for example), then the `render_list_style` option would become `render_list_style_dash`, `render_list_style_star`, `render_list_style_plus`.... see my point?

I move to drop both the `extension_wikilinks_title_before_pipe` and `extension_wikilinks_title_after_pipe` option in favor of a new `extension_wikilinks_title` option, which could support enumerated values (either string or numeric) for before, after, or undefined (e.g. default).